pvlib/pvsystem.py

```diff
@@ -1011,28 +1040,21 @@ def calcparams_desoto(effective_irradiance, temp_cell,
     Tuple of the following results:
 
     photocurrent : numeric
-        Light-generated current in amperes at irradiance=S and
-        cell temperature=Tcell.
+        Light-generated current in amperes
 
     saturation_current : numeric
-        Diode saturation curent in amperes at irradiance
-        S and cell temperature Tcell.
+        Diode saturation curent in amperes
 
     resistance_series : float
-        Series resistance in ohms at irradiance S and cell temperature
-        Tcell.
+        Series resistance in ohms
```

Comment:
While passing through this territory, seems like this should be changed to "numeric" from "float".

Reply:
Cut and paste inheritance?  I don't think there is a difference here. `numeric` would admit `long`, `complex` and `int` types as well as `float`.  Probably should change to `float` unless there's a reason otherwise.

Reply:
I'm good with float instead of numeric, because that seems to be most accurate.